backend/src/agents/maths_agent.py

```diff
@@ -0,0 +1,47 @@
+from .tool import tool_metadata
+from .types import Parameter
+from .agent import Agent, agent_metadata
+
+
+@tool_metadata(
+    name="compare two values",
+    description="Compare two passed values and return information on which one is greater",
+    parameters={
+        "thing_one": Parameter(
+            type="string",
+            description="first thing for comparison",
+        ),
+        "value_one": Parameter(
+            type="number",
+            description="value of first thing",
+        ),
+        "thing_two": Parameter(
+            type="string",
+            description="second thing for comparison",
+        ),
+        "value_two": Parameter(
+            type="number",
+            description="value of first thing",
+        )
+    },
+)
+def compare_two_values(value_one, thing_one, value_two, thing_two) -> str:
+    if (value_one > value_two):
+        return f"You have spent more on {thing_one} ({value_one}) than {thing_two} ({value_two}) in the last month"
+    else:
+        return f"You have spent more on {thing_two} ({value_two}) than {thing_one} ({value_one}) in the last month"
```

Comment:
can we drop `in the last month`? (here and line above)

I believe this code is meant to be generic so can compare two values over any time period.

Reply:
We can, but it is also still very much a stub response and should be treated as one. I'm not a huge fan of half converting this method into a working comparison method. Would rather refactor the `MathsAgent` all in 1 go to no longer be a stub than tweak this part

Reply:
Nor me.... sorry I didn't realise we still need this to be a stub.

I assumed converting this to a real implementation would be fine as it's using values being passed-in. I assumed only the `datastore_agent` needed stubs as we're only missing the database.

If you want to roll this back to be a 100% stub, please do.